docs/fastdds/xml_configuration/topic.rst

```diff
@@ -35,7 +35,8 @@ The ``<topic>`` element has two attributes defined: ``profile_name`` and ``is_de
 Topic configuration
 ^^^^^^^^^^^^^^^^^^^
 
-This XML element allows the configuration of the :ref:`historyqospolicy` and :ref:`resourcelimitsqospolicy`.
+This XML element allows the configuration of the :ref:`historyqospolicy` and :ref:`resourcelimitsqospolicy` as
+the default QoS for any entity that contains those policies.
```

Comment:
I think it is better to just mention that allows the configuration of those QoS in the Topic and a link to the TopicQos section in any case, because we are not really sure how the TopicQoS influences the endpoints created on them.

Reply:
I have a couple of things related to this change.

1. You have linked to the Common TopicType XML element. There it is mentioned that this element configures the HistoryQoS and ResourceLimitsQos for DataWriters and DataReaders. This is no longer true because the TopicQos is also configured (with the topic profile).
2. My intention was that you linked directly with [TopicQos](https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/topic/topic/topic.html#topicqos), not with [TopicType](https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/common.html#topictype)